pkgs/applications/version-management/forgejo/default.nix

```diff
@@ -1,25 +1,66 @@
-{ fetchurl, gitea, lib }:
+{ lib
+, stdenv
+, buildGoPackage
+, fetchurl
+, makeWrapper
+, git
+, bash
+, gzip
+, openssh
+, pam
+, pamSupport ? true
+, sqliteSupport ? true
+}:
 
-gitea.overrideAttrs (old: rec {
+buildGoPackage rec {
   pname = "forgejo";
-  version = "1.18.0-rc1-1";
+  version = "1.18.2-0";
```

Comment:
If the suffix is always -0 we should drop it in the future.

Reply:
Hm... This is correct version:
```
./result/bin/gitea --version
Gitea version 1.18.2-0 built with go1.19.4 : pam, sqlite, sqlite_unlock_notify
```

Reply:
We are hardcoding the version in line 43, so I am not surprised that it outputs this.

Reply:
The dash one is sometimes used, so we keep it https://codeberg.org/forgejo/-/packages/container/forgejo/1.18.2-1

Reply:
Yeah, just for future reference, the versioning is explained in
https://codeberg.org/forgejo/forgejo/src/branch/forgejo/CONTRIBUTING/RELEASE.md#release-numbering [[permalink](https://codeberg.org/forgejo/forgejo/src/commit/0db7131d5bd07a2c8b7874457998551fd601c6c4/CONTRIBUTING/RELEASE.md#release-numbering)]